src/index.ts

```diff
@@ -0,0 +1,9 @@
+export {
+  Parser,
+  ParseResult,
+  ContractMetadata,
+  fetchContractSchemasBytes,
+} from './parser/parser';
```

Comment:
Let's move content of `parser` directly into `src`. The extra nesting seems to be redundant.